vncviewer/GestureHandler.cxx

```diff
@@ -41,10 +42,11 @@ int GestureHandler::registerEvent(void *ev) {
       // FIXME: We can distinguish between different two-touch gestures
       // by looking at relative and absolute movement. This logic goes here.
       // For now, just pass the update on to be processed normally.
-      processed.push_back({0,
-		           devev->event_x,
-		           devev->event_y,
-			   GH_TouchUpdate});
+      ghev.detail = 0;
+      ghev.event_x = devev->event_x;
+      ghev.event_y = devev->event_y;
+      ghev.type = GH_TouchUpdate;
+      processed.push_back(ghev);
```

Comment:
You could always create a method that takes those for values as arguments. Then you'd get something very similar to your original `push_back()`.

Reply:
Yeah, this sort of thing is going to be happening all over the place, so I plan to provide some kind of short-hand for it in due course. Point taken.